### DISCUSSION TITLE
Is there any way to figure out what colors can be removed from the config?

### DISCUSSION THREAD

**Discussion by user753** [tailwindlabs/tailwindcss#4541] [Help]
I have many colors in my config and I want to know what colors are used in the project and what colors can be removed?

**Comment by devntv** [comment]
Tailwindcss provides color codes available per class.
if you want to extend color code just add `extend`, else remove `extend` if you want to override all default color codes.
The codes used are the ones included in the config
documentation: https://tailwindcss.com/docs/customizing-colors#default-color-palette

**Comment by olets** [comment]
You can track down every use of a color, with global searches for `text-`, `bg-`, etc.

Or the inverse: run a production build, remove a color, run a production build, and compare the stylesheets in the two builds. If removing the color doesn't change the build, it wasn't being used.

**Reply by user753** [reply]
>Or the inverse: run a production build, remove a color, run a production build, and compare the stylesheets in the two builds. If removing the color doesn't change the build, it wasn't being used.

It's what I use now, but I hope there would be some tool to do it automaticly.
